shell/common/persistent_cache.h

```diff
@@ -17,6 +17,7 @@
 
 namespace flutter {
 
+/// A cache of shaders that gets stored to disk.
```

Comment:
Doesn't only have to be shaders. Also, all stores are atomic and serviced on an optional worker thread.

Reply:
The files saved to disk are named "shader_dump_x.skp" https://github.com/flutter/engine/blob/7bb5b9aea51c66645407903ec55004a40a35c65f/shell/common/persistent_cache.cc#L172

Also store always sets "stored_new_shaders_ = true;"

If it stores more than shaders there is other code in here that is out of date.

Reply:
> Doesn't only **have** to be shaders.

Its just shaders for now. But the cache directly is also written to by Dart. If you think this mentioning shaders is clearer, then go for it.

Reply:
Done.  Added Shaders in details and out of brief.